internal/cmd/build_helper.go

```diff
@@ -132,11 +132,14 @@ func newFoundry(ctx context.Context, opts *buildOptions) (k6foundry.Foundry, err
 			CopyGoEnv: true,
 			Env:       env,
 		},
-		K6Repo:      opts.k6repo,
 		SkipCleanup: opts.skipCleanup != 0,
 		Logger:      logger,
 	}
 
+	if opts.k6repo != defaultK6Repo { // only set if different from default: k6foundry workaround
```

Comment:
Alternatively, we could make the default **k6 replacement** repository (this is how the parameter is defined in `k6foundry`) as `""`, unless there's a reason for using k6's repo as a default replacement repo. Then this won't be necessary.